tests/pages/api/priceEntry.test.ts

```diff
@@ -1,66 +1,77 @@
-import priceEntry, { numberOfInputsIsValid, putDataInS3 } from '../../../src/pages/api/priceEntry';
+import * as priceEntryApi from '../../../src/pages/api/priceEntry';
+import priceEntry from '../../../src/pages/api/priceEntry';
 import { getMockRequestAndResponse } from '../../testData/mockData';
 
 describe('Price Entry API', () => {
     describe('API validation of number of price inputs', () => {
```

Comment:
Could we change these describes to be more consistent with the other tests? (i.e. just the name of the page/api/function they are testing)